EIPS/eip-5131.md

```diff
@@ -0,0 +1,294 @@
+
+### Authenticating `mainAddress` via `authAddress`
+Control of `mainAddress` and ownership of `mainAddress` assets is proven if any one of associated `authAddress` is the msg.sender or has signed the message.
+
+Practically, this would work by performing the following operations:
+1. Get the reverse ENS record for `authAddress`
+2. Parse `auth[0-9A-Za-z]*.<mainENS>` to determine the linked ENS
+3. Do a lookup on the linked ENS record to determine the linked `mainAddress`
+4. MUST get the reverse ENS record for `mainAddress` and verify that it matches `<mainENS>` 
+    - Otherwise one could set up other ENS nodes (with auths) that point to `mainAddress` and authenticate via those.
+
+Note that this specification allows for both contract level and client/server side validation of signatures.  It is not limited to smart contracts, which is why there is no proposed external interface definition.
+
+## Rationale
+The proposed specification allows one to link multiple addresses as 'authentication addresses' to a core main address. This is beneficial from a security standpoint (if the authentication address is compromised, the assets held by the main address is not), and convenience (if the authentication address is a simple MetaMask wallet but the main address is a hardware wallet).
+
+### Example 1: Event Access
+An event requires attendees to prove ownership of an NFT to gain access. The attendee in question only has their phone available to perform a signing operation upon the presentation of a challenge. The phone is considered the attendees 'hot wallet' which is linked to auth.wilkins.eth.
+The NFT is owned by the Ethereum address that corresponds to wilkins.eth, which is controlled by the attendees' cold storage device (Ledger, Trezor etc). The attendee does not have this device present at the event.
+The attendee is able to prove ownership of the NFT by signing with the 'hot wallet' key.
+
+This is made possible because the attendee completed the necessary upfront authorization steps with the cold storage device to allow the 'hot wallet' to be used for this operation. 
+
+### Example 2: Multiple Devices
+I have multiple devices and wish to sign with any of them for convenience.  In this situation:
+ - My phone has mobile metamask hot wallet, and I can add it as an auth account by setting auth1.wilkins.eth to that wallet (and the corresponding reverse record). 
+ - My tablet has the same and can be set to auth2.wilkins.eth, etc. 
+
+Lost Access to Signing Key/Device: 
+In the event that a user losses access to the signing key of an `authAddress` they can simply delete the corresponding ENS record. An attractive side effect of this is that there is no need to import/reimport seed phrases. 
+In the unfortunate event that you lose access to the signing key of the `mainAddress`, users will lose access to that top-level domain and all related subdomains. As always, this private key data should be treated as highly sensitive. 
```

Comment:
This section still reads as Motivation, and not Rationale. Motivation should explain why the EIP needs to exist as a whole, while the Rationale should explain individual technical choices within the EIP itself. For example, your Rationale section could explain why you don't limit the number of `authAddress` entries (though I'm not saying you _have_ to explain that particular choice.)